ui/components/app/hold-to-reveal-button/index.scss

```diff
@@ -36,8 +37,7 @@ $circle-stroke-width: 2px;
   }
 }
 
-.hold-to-reveal-button {
-  // Shared styles
+%hold-to-reveal-button {
```

Comment:
stylelint now complains if `@extend` isn't used with a `%`-selector, so I've moved some things in this file.